<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgb(3, 4, 19);
border: 2px solid rgb(121, 121, 121);
}</string>
  </property>
  <widget class="QPushButton" name="NextBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color:  #252984;
color: white;
font: bold 25px;</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color:  #252984;
color: white;
font: bold 25px;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="WidthLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	color: white;
	font: bold 25px;
}</string>
   </property>
   <property name="text">
    <string>Stock width</string>
   </property>
  </widget>
  <widget class="QLabel" name="NumLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	color: white;
	font: bold 25px;
}</string>
   </property>
   <property name="text">
    <string>Number of rectangles</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="WidthSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background: white;
    border: 3px solid #797979;
	font: bold 25px;
	padding-left: 35px;
	color: black;
}

QSpinBox::up-button {
    width: 0;
	height: 0;
}
QSpinBox::down-button {
    width: 0;
	height: 0;
}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background: white;
	font: bold 25px;
    border: 3px solid #797979;
	padding-left: 35px;
	color: black;
}

QSpinBox::up-button {
    width: 0;
	height: 0;
}
QSpinBox::down-button {
    width: 0;
	height: 0;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
